.github/workflows/default.yml

```
@@ -13,10 +13,10 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v3
 
-      - name: nodejs setup (12.x)
+      - name: nodejs setup (24)
         uses: actions/setup-node@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)